packages/rum-core/src/common/config-service.js

```diff
@@ -154,8 +154,13 @@ class Config {
   }
 
   getEndpointUrl() {
-    var url = this.get('serverUrl') + this.get('serverUrlPrefix')
-    return url
+    /**
+     * Remove all trailing slash for serverUrl since serverUrlPrefix
+     * includes a forward slash for the path
+     */
+    this.config.serverUrl = this.config.serverUrl.replace(/\/+$/, '')
```

Comment:
I don't think this is also a good place to do it, we're creating side-effect in a method that is supposed to only 'get' the url. I think the best place to do it is on setConfig maybe we can have two methods one for validation and one for sanitising values

Reply:
Hmmm, setConfig is super generic.  We have to do this transformation only once and not every-time when user sets a config. Let me know where you think would be the best place.

Reply:
Decided to fix the `serverUrl` in setConfig before replacing the configuration.